unstructured/ingest/v2/processes/partitioner.py

```diff
@@ -116,14 +116,25 @@ def create_partition_parameters(self, filename: Path) -> "PartitionParameters":
         from unstructured_client.models.shared import Files, PartitionParameters
 
         partition_request = self.config.to_partition_kwargs()
+        possible_fields = [f.name for f in fields(PartitionParameters)]
+        filtered_partition_request = {
+            k: v for k, v in partition_request.items() if k in possible_fields
+        }
+        if len(filtered_partition_request) == len(partition_request):
```

Comment:
I don't understand this. I ran a test with 
--strategy fast \
--partition-by-api \

And got this:
`Following fields were omitted due to not being supported by the currently used unstructured client: `
with nothing past the `:`

Reply:
Had the boolean logic wrong, updated it from `==` to `!=` so this shouldn't show up anymore if no fields were omitted.